tests/src/JIT/opt/Tailcall/TailcallVerifyWithPrefix.il

```diff
@@ -8795,15 +8797,17 @@
       IL_003a:  call       void [System.Console]System.Console::WriteLine(string)
       IL_003f:  ldstr      "v.i1: {0} != byte.MinValue || v.i2: {1} != short.M"
```

Comment:
Previously this was using string formatting to substitute {0} and {1},
Your expansion here result is a different output,
There should be a CoreCLR equivalent way to print formatted strings.